src/lib/storage/snapshots_finish.rb

```diff
@@ -0,0 +1,48 @@
+require "yast2/fs_snapshot"
+require "installation/finish_client"
+
+module Storage
```

Comment:
There is already storage as libstorage ruby bindings. Are you sure it do not collide?